prime-router/src/main/kotlin/JurisdictionalFilters.kt

```diff
@@ -300,6 +371,8 @@ object JurisdictionalFilters: Logging {
         "hasAtLeastOneOf(order_test_date,specimen_collection_date_time,test_result_date)",
         // has at least one valid CLIA
         "isValidCLIA(testing_lab_clia,reporting_facility_clia)",
+        // never send T (Training/Test) or D (Debug) data to the states.
+        // NOT TURNED ON YET    "doesNotMatch(processing_mode_code,T,D)",
```

Comment:
This needs more testing, and I think I need to implement a different default filter in Local/Staging vs Prod, before going live with this.